static/app/utils/useResizableDrawer.tsx

```diff
@@ -85,14 +65,11 @@ export function useResizableDrawer(options: UseResizableDrawerOptions): {
   // any potentional values set by CSS will be overriden. If no initialDimensions are provided,
   // invoke the onResize callback with the previously stored dimensions.
   useLayoutEffect(() => {
-    options.onResize(options.initialSize ?? 0, size, false);
-    setSize(options.initialSize ?? 0);
+    sizeRef.current = options.initialSize;
+    options.onResize(options.initialSize ?? 0, false);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [options.direction]);
```

Comment:
the only thing we exclude here is `options`, but if options were put into a ref with the pattern I outlined before, we wouldn’t need to disable the linter here.